src/btree/bt_import.c

```diff
@@ -137,7 +137,7 @@ __wt_import_repair(WT_SESSION_IMPL *session, const char *uri, char **configp)
 err:
     F_CLR(session, WT_SESSION_IMPORT_REPAIR);
 
-    __wt_meta_ckptlist_free(session, &ckptbase);
+    __wt_checkpoint_ckptlist_free(session, &ckptbase);
```

Comment:
This is a bit verbose, what about `__wt_checkpoint_list_free`? But "list" is too general.
What are your thoughts on this, @sueloverso?

Reply:
My proposal is to consistently use the same word for terms. If "checkpoint list" is a term then it'd be good to agree on what keyword we use for this. `ckptlist` looks ok to me but it could be something else.